docs/quickstart.rst

```diff
@@ -0,0 +1,385 @@
+
+Quickstart
+==========
+
+Installation
+------------
+
+pyrefinebio can be installed through PyPI:
+
+.. code-block:: shell
+
+   $ pip install pyrefinebio
+
+
+Setting up Tokens
+-----------------
+
+Before you start downloading or interacting with the API you should set up a Token.
+Tokens will allow you to be able to download Compendia, Computed Files, and Datasets.
+
+Creating and activating a refine.bio Token indicates agreement with refine.bio's `Terms of Use`_ and
+`Privacy Policy`_ so make sure you have read and understood both before you continue.
+
+.. _Terms of Use: https://www.refine.bio/terms
+.. _Privacy Policy: https://www.refine.bio/privacy
+
+To create a token, make an object of the Token class:
+
+.. code-block:: python
+
+    import pyrefinebio
+
+    token = pyrefinebio.Token()
+
+Then, to activate your token, call `agree_to_terms_and_conditions()` on it:
+
+.. code-block:: python
+
+    token.agree_to_terms_and_conditions()
+
+Now, your Token is fully set up and will be used in any API requests made during the execution of this script.
+
+If you want to use the same Token in future scripts, you can save it to the Config file:
+
+.. code-block:: python
+
+    token.save_token()
+
+Now, the token will be automatically loaded every time you import pyrefinebio.
+
+If you want to get access to the token that is saved to the config file in the future, you can load it:
+
+.. code-block:: python
+
+    token = pyrefinebio.Token.load_token()
+
+For more information check out the :ref:`Token` class documentation and :ref:`Config`.
+
+Downloading Datasets
+--------------------
+
+After you set up and activate a Token you can start creating and downloading Datasets.
+
+pyrefinebio provides the function `download_dataset()` that makes creating and downloading Datasets super easy.
```

Comment:
```suggestion
pyrefinebio provides the function `download_dataset()` for creating and downloading Datasets.
```
I think it's good to avoid claiming anything is easy in docs in case whoever is reading them is currently struggling with it.